includes/template-functions.php

```diff
@@ -1128,6 +1128,8 @@ function edd_pagination( $args = array() ) {
 	);
 
 	$args = wp_parse_args( $args, $defaults );
+	
+	$args = apply_filters( 'edd_pagination_args', $args );
```

Comment:
Can you add some PHPDocs for this please? Here's an example of how they're formatted: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/privacy-functions.php#L881-L889 Just document the parameter and `@since 3.0`